.github/workflows/check-dist.yml

```diff
@@ -21,12 +21,12 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
 
-      - name: Set Node.js 16.x
-        uses: actions/setup-node@v2.5.1
+      - name: Set Node.js
+        uses: actions/setup-node@v4
```

Comment:
The joys of maintaining github actions 😉

Reply:
You there?? Now it will be dev and js dependencies... 😇